src/runtime/job.cpp

```diff
@@ -824,8 +834,28 @@ static void launch(JobTable *jobtable) {
     } else {
       err = std::make_unique<NullBuf>();
     }
+
+    auto stdout_teefiles = split_null(task.job->stdout_teefiles);
+    auto stdout_tee_names = std::vector<std::string>();
+    for (int i = 0; stdout_teefiles[i]; ++i) {
+      stdout_tee_names.push_back(stdout_teefiles[i]);
+      if (jobtable->imp->teefiles.find(stdout_teefiles[i]) == jobtable->imp->teefiles.end()) {
+        jobtable->imp->teefiles[stdout_teefiles[i]] = std::make_unique<std::ofstream>(stdout_teefiles[i], std::ios::out | std::ios::trunc);
```

Comment:
We should do an unlink after this so that we get a new inode instead of modifying the existing file.

Reply:
I'm not seeing that function in the C++ reference I'm using, though I do recognize what it means in context.  Mind giving me a bit more guidance what that would look like in code?

Reply:
unlink is a posix function: https://man7.org/linux/man-pages/man2/unlink.2.html

Reply:
Apparently it's a bit less complete a reference than I expected, thanks!  (They could at least have included it and marked it as system dependent somehow...)

Reply:
```
unlink(file.c_str());
```

Is all you would need but its probably best to only ignore the error if ENOENT is returned. Everything else should be logged as a warning


```
if (unlink(file.c_str()) < 0 && errno != ENOENT) {
    wcl::log::warning("Could not unlink %s while creating a new file for tee-ing: %s", file.c_str(), strerror(errno))();
}
```